src/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -721,33 +721,125 @@ public String getID() {
     // get routines to access context info fields
     //
     // ---------------------------------------------------------------------------------------------------------
+    
+    /** shortcut to {@link CommonInfo#getName()} 
+     * @return the source name
+     */
     public String getSource()                   { return commonInfo.getName(); }
+    
+    /** shortcut to {@link CommonInfo#getFiltersMaybeNull()} 
+     *  @return a <b>modifiable</b> Set of filters. Can be  null. All changes in this set will be reflected in the CommonInfo 
+     */
     public Set<String> getFiltersMaybeNull()    { return commonInfo.getFiltersMaybeNull(); }
+    
+    /** shortcut to {@link CommonInfo#getFilters()} 
+     * @return an unmodifiable Set of filters. Can be empty by never null
+     */
     public Set<String> getFilters()             { return commonInfo.getFilters(); }
+    
+    /** shortcut to {@link CommonInfo#isFiltered()}
+     * @return true if any filter been defined  <code>(!filters.isEmpty())
+     */
     public boolean isFiltered()                 { return commonInfo.isFiltered(); }
+    
+    /** shortcut to {@link CommonInfo#isNotFiltered()} */
     public boolean isNotFiltered()              { return commonInfo.isNotFiltered(); }
+    
+    /** shortcut to {@link CommonInfo#filtersWereApplied()}
+     *  @return true if no filter has been defined  <code>(getFiltersMaybeNull()==null)</code>
+     */
     public boolean filtersWereApplied()         { return commonInfo.filtersWereApplied(); }
+    
+    /** shortcut to {@link CommonInfo#hasLog10PError()}
+     * @return true if log10-based error estimate has been set 
+     */
     public boolean hasLog10PError()             { return commonInfo.hasLog10PError(); }
+    
+    /** shortcut to {@link CommonInfo#getLog10PError()}
+     * @return the -1 * log10-based error estimate
```

Comment:
This isn't right -- it's just log10(perror), not -1 * log10(perror). You can see this is true because in `commonInfo.getPhredScaledQual()` we multiply `getLog10PError()` by -10.